src/features/session_replay/aggregate/index.js

```diff
@@ -67,6 +67,8 @@ let recorder, gzipper, u8
 export const MAX_PAYLOAD_SIZE = 1000000
 /** Unloading caps around 64kb */
 export const IDEAL_PAYLOAD_SIZE = 64000
+/** Reserved room for query param attrs */
+const QUERY_PARAM_PADDING = 5000
```

Comment:
Just curious, how much room does this leave us in the required params section?

Reply:
Yea I guess what I am asking is how much of the 5kb are we currently consuming with the required attributes.

Reply:
right now the req'd params encompass about 0.5kb